h2o-hadoop/h2o-mapreduce-generic/src/main/java/water/hadoop/h2odriver.java

```diff
@@ -1537,9 +1534,16 @@ private int run2(String[] args) throws Exception {
     job.setOutputValueClass(Text.class);
 
     FileInputFormat.addInputPath(job, new Path("ignored"));
-    if (outputPath != null) {
-      FileOutputFormat.setOutputPath(job, new Path(outputPath));
+
```

Comment:
There seems to be a simpler solution. You can disable FileOutputFormat and use NullOutputFormat:

    if (outputPath != null)
        FileOutputFormat.setOutputPath(job, new Path(outputPath));
    else
        job.setOutputFormatClass(NullOutputFormat.class);

Then you don't need to create the temporary directories.